src/site/content/en/handbook/inclusion-and-accessibility/index.md

```diff
@@ -5,20 +5,19 @@ authors:
   - mfriesenhahn
 date: 2019-06-26
 description: |
-  This handbook helps contributors to web.dev create effective, engaging content
-  and get it published as easily as possible.
+  Learn how to write for an international audience.
```

Comment:
Or some such description. This looks like a copy/paste error.

Reply:
Yes, copy/paste error. Thanks!